wolox-books/src/app/screens/list-books/components/cards/cards.component.html

```diff
@@ -0,0 +1,5 @@
+<div class="content-card" *ngFor="let item of books">
```

Comment:
The component you created should be only the card, not the creation of multiple cards